### PR TITLE
[VEN789] [VEN-747] docs: Edit oracle doc descriptions

### PR DESCRIPTION
**Pull request by coreyar**
Edits Oracle doc introduction and details for clarity.

Adds most recent nat spec

### CODE REVIEW

File: oracles/binance.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This oracle fetchs prices of assets from Binance oracle.
+This oracle fetchs prices of assets from the Binance oracle.
```
Comment (0xlucian) [inline oracles/binance.md:5]:
```suggestion
This oracle fetches prices of assets from the Binance oracle.
```

File: oracles/chainlink.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This oracle fetchs prices of assets from Chainlink oracle.
+This oracle fetchs prices of assets from the Chainlink oracle.
```
Comment (0xlucian) [inline oracles/chainlink.md:5]:
```suggestion
This oracle fetches prices of assets from the Chainlink oracle.
```

File: oracles/pyth.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This oracle fetches price of assets from Pyth oracle
+This oracle fetches price of assets from the Pyth oracle
```
Comment (0xlucian) [inline oracles/pyth.md:5]:
```suggestion
This oracle fetches price of assets from the Pyth oracle.
```

**Review by 0xlucian — CHANGES_REQUESTED** [review]

**Comment by chechu** [comment]
Included in https://github.com/VenusProtocol/venus-protocol-documentation/pull/35

**Pull request closed without merging**